src/resources/app/components/AnnouncementCard.vue

```diff
@@ -6,10 +6,11 @@
   >
     <div id="announcement">
       <AnnouncementContent
-        :name=this.$props.createdBy
+        :name=this.$props.author
```

Comment:
Just wanna ask, what's the difference between this.$props.author and "this.$props.author" ?